src/python/dxpy/templating/templates/nextflow/src/nextflow.sh

```diff
@@ -365,7 +367,7 @@ main() {
     $RESUME_CMD \
     $nextflow_run_opts \
     $nextflow_pipeline_params \
-    @@REQUIRED_RUNTIME_PARAMS@@
+    $required_inputs
```

Comment:
need to work together with
https://github.com/dnanexus/dx-toolkit/blob/master/src/python/dxpy/nextflow/nextflow_templates.py#L83-L87